src/Symfony/Component/Yaml/Tests/DumperTest.php

```diff
@@ -913,7 +913,7 @@ public function testDumpNullAsTilde()
     /**
      * @dataProvider getForceQuotesOnValuesData
      */
-    public function testCanForceQuotesOnValues(array $input, string $expected)
+    public static function testCanForceQuotesOnValues(array $input, string $expected)
```

Comment:
this function is the test, not the data provider

Reply:
oops 🙈 thanks for the heads up